Libraries/Opc.Ua.Server/Diagnostics/DiagnosticsNodeManager.cs

```diff
@@ -1354,99 +1354,30 @@ private bool UpdateServerDiagnosticsSummary()
                             HasApplicationSecureAdminAccess(context);
             }
 
+            IEnumerable<FieldInfo> roleIds = typeof(ObjectIds).GetFields().Where(predicate: fieldInfo => {
+                return fieldInfo.Name.StartsWith("WellKnownRole", StringComparison.InvariantCulture);
+            });
+
```

Comment:
how often is this overload called? If its only once its ok, but if this is called frequently the query on all objectids may add a lot of overhead. I suggest in that case to write out the small array with all well known Ids.

 Second storing just the NodeId (say return GetValue(null)) it becomes a NodeIdCollection which is quite more handy to use?